### DISCUSSION TITLE
Can't change DNS listen port

### DISCUSSION THREAD

**Discussion by bennetluedersen** [crazy-max/docker-cloudflared#99] [Q&A] [answered]
### Behaviour
Can't change DNS listen port
#### Steps to reproduce this issue

1. Use UnRaid
2. Get the container through Community Applications
3. In the settings for the container set the DNS listen port to any port other than 5053

#### Expected behaviour

The container should change its DNS listen port accordingly to the port set in the configuration.

#### Actual behaviour
The container always uses port 5053.

### Configuration

* Docker version 19.03.5, build 633a0ea838
* Docker compose version if applicable (type `docker-compose --version`) : 
* Platform: UnRaid 6.8.3
* System info (type `uname -a`) : Linux Tower 4.19.107-Unraid #1 SMP Thu Mar 5 13:55:57 PST 2020 x86_64 Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz GenuineIntel GNU/Linux
* Include all necessary configuration files : 

### Docker info

Client:
 Debug Mode: false

Server:
 Containers: 17
  Running: 12
  Paused: 0
  Stopped: 5
 Images: 17
 Server Version: 19.03.5
 Storage Driver: btrfs
  Build Version: Btrfs v4.7.3
  Library Version: 101
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: b34a5c8af56e510852c35414db4c1f4fa6172339
 runc version: 3e425f80a8c931f88e6d94a8c831b9d5aa481657
 init version: fec3683
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 4.19.107-Unraid
 Operating System: Slackware 14.2 x86_64 (post 14.2 -current)
 OSType: linux
 Architecture: x86_64
 CPUs: 32
 Total Memory: 31.43GiB
 Name: Tower
 ID: GWTT:CIR3:JXTA:XM6F:O45K:DW5E:APOI:CM7N:VX5B:DXFI:7GM6:WL34
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Username: lustteufel
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
 Product License: Community Engine

### Logs

```
> Container logs (set LOG_LEVEL to debug if applicable)
```

**Comment by crazy-max** [comment, marked as answer]
@bennetluedersen

> port to any port other than 5053

You can already map port 5053 in the container to any port on the Docker host with `-p 53:5053`.
See also https://docs.docker.com/config/containers/container-networking/#published-ports
